build.sbt

```diff
@@ -0,0 +1,188 @@
+import akka._
+import com.typesafe.sbt.SbtScalariform.ScalariformKeys
+import com.lightbend.paradox.sbt.ParadoxPlugin
+import akka.AkkaDependency.RichProject
+import com.typesafe.sbt.packager.docker._
+import com.typesafe.sbt.SbtMultiJvm.MultiJvmKeys.MultiJvm
+import Dependencies.{ Scala211, Scala212, Scala213, buildScalaVersion }
+
+
+lazy val root = Project(id = "akka-enhancements", file("."))
+  .settings(defaultSettings)
+  .dependsOn(diagnostics)
+  .enablePlugins(UnidocRoot, NoPublish)
+  .settings(
+      UnidocRoot.autoImport.unidocRootIgnoreProjects := List(
+          docs,
+          addonsTestkit),
+  )
+
+lazy val diagnostics = akkaAddonsModule("akka-diagnostics")
+  .settings(
+    Dependencies.akkaDiagnostics)
+  .settings(Release.settings: _*)
+  .enablePlugins(BootstrapGenjavadoc)
+  .dependsOn(
+    addonsTestkit % "test",
+    // so that the tests can check the sbr config
+    splitBrainResolver % "test")
+  .addAkkaModuleDependency("akka-actor")
+  // used for verifying the config checker
+  .addAkkaModuleDependency("akka-remote", "test")
+  .addAkkaModuleDependency("akka-cluster", "test")
+  .addAkkaModuleDependency("akka-cluster-metrics", "test")
+  .addAkkaModuleDependency("akka-cluster-sharding", "test")
+  .addAkkaModuleDependency("akka-cluster-tools", "test")
+  .addAkkaModuleDependency("akka-persistence", "test")
+
+lazy val splitBrainResolver = akkaAddonsModule("akka-split-brain-resolver")
```

Comment:
I have this comment in `akka-diagnostics` 
```
 .dependsOn(
    addonsTestkit % "test",
    // so that the tests can check the sbr config
    splitBrainResolver % "test")
```
Shall we not be testing the SBR config?

Reply:
Never mind. It seems this dependency is not used. The 24 previous tests pass either way with out this.